### ISSUE TITLE
Connection error. Please try again: TypeError: Failed to fetch

### ISSUE THREAD

**Issue by Screwyparasite** [Pushan2005/SpotTransfer#2]
Error Pop up when selecting connect.  

All copied from Accept */* through the end. Worked initially but will not work now.

Developers tools produce the following errors:

"create-playlist:1 Access to fetch at 'https://spottransfer-backend.onrender.com/' from origin 'https://spot-transfer.vercel.app' has been blocked by CORS policy: Response to preflight request doesn't pass access control check: No 'Access-Control-Allow-Origin' header is present on the requested resource. If an opaque response serves your needs, set the request's mode to 'no-cors' to fetch the resource with CORS disabled."

&

"GET https://spottransfer-backend.onrender.com/ net::ERR_FAILED
R @ index-B54dbQzw.js:101
test.o.<computed> @ index-B54dbQzw.js:51
n0 @ index-B54dbQzw.js:37
o0 @ index-B54dbQzw.js:37
i0 @ index-B54dbQzw.js:37
af @ index-B54dbQzw.js:37
lf @ index-B54dbQzw.js:37
(anonymous) @ index-B54dbQzw.js:37
Hl @ index-B54dbQzw.js:40
hd @ index-B54dbQzw.js:37
Ha @ index-B54dbQzw.js:37
Ra @ index-B54dbQzw.js:37
S0 @ index-B54dbQzw.js:37"

![image](https://github.com/user-attachments/assets/e2ed28a8-de1a-44b5-926e-2accbcef1eb9)

**Comment by Jayc965** [comment]
Same error, when clicked on '**Connect**' for the option:''**You need to be connected to the server**'.
![image](https://github.com/user-attachments/assets/5a288f20-2ed3-4618-8cbc-f29a85656d06)

**Comment by ameenalasady** [comment]
Duplicate of issue #1. I'm guessing backend service got suspended again

**Comment by Pushan2005** [comment]
@ameenalasady you're right. I'll have it up soon

**Comment by Screwyparasite** [comment]
Just a heads up, it happened again now

**Comment by Pushan2005** [comment]
@Screwyparasite thanks for letting me know, should be back up soon
